src/app/(proper_react)/(redesign)/(public)/breaches/page.tsx

```diff
@@ -30,6 +30,9 @@ export function generateMetadata() {
 
 export default async function Page() {
   const allBreaches = await getBreaches();
+  const breachesNewestFirst = allBreaches.sort((a, b) => {
+    return +new Date(b.BreachDate) - +new Date(a.BreachDate);
```

Comment:
Oh actually, the acceptance criteria aren't clear, but we might want to show the "newest" breaches in the sense of most recently added - sometimes they only get added years after they happen.

```suggestion
    return +new Date(b.AddedDate) - +new Date(a.AddedDate);
```